assets/shaders/animate_shader.wgsl

```diff
@@ -1,39 +1,7 @@
 #import bevy_pbr::mesh_types
+// The time since startup data is in the globals binding which is part of the mesh_view_bindings import
 #import bevy_pbr::mesh_view_bindings
```

Comment:
What about 2d users? They may not depend on `bevy_pbr`, but it's possible they still want to access the globals binding. 

Possible solutions (imo):
* make the globals binding available also in `bevy_sprite::mesh2d_view_bindings`
* have an example with sprites that shows importing `mesh_view_bindings` from `bevy_pbr`

Reply:
Good point, I'll address this in a follow up PR. I honestly didn't think of that. I'll probably add the bindings to `mesh2d_view_bindings`. Although, I wonder if we could have some sort of "core" bindings that are shared between both.